hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -1660,36 +1660,31 @@
       </description>
   </property>
 
-  <property>
-    <name>dfs.block.report.load.balancing.time.window.size</name>
-    <value>60000</value>
-    <description>
-      Window of time to compute NN threashold for block reporting
-    </description>
-  </property>
-  
   <property>
     <name>dfs.block.report.load.balancing.db.var.update.threashold</name>
     <value>60000</value>
     <description>
-      Number of blocks that can be reported to a NN during the load balancing 
+        Number of concurrent block reports processed by the namenodes. 
```

Comment:
Shouldn't it be something "update frequency of the variable indicating the number of concurrent block reports processed by the namenodes (for online update purpose)."

Reply:
fixed